cmd/introspect/baml_parser_golden_test.go

```
@@ -0,0 +1,413 @@
+				AccessKeyID:     snapshotBedrockOptionValue(opts.Credentials.AccessKeyID),
+				SecretAccessKey: snapshotBedrockOptionValue(opts.Credentials.SecretAccessKey),
+				SessionToken:    snapshotBedrockOptionValue(opts.Credentials.SessionToken),
+				Profile:         snapshotBedrockOptionValue(opts.Credentials.Profile),
+			},
+		}
+	}
+	return snap
+}
+
+func snapshotBedrockOptionValue(v bedrockOptionValue) bedrockOptionValueSnapshot {
+	return bedrockOptionValueSnapshot{
```

golangci-lint (2.12.2)

[error] S1016: should convert v (type bedrockOptionValue) to bedrockOptionValueSnapshot instead of using struct literal

(staticcheck)